android/src/main/java/com/mattermost/turbolog/TurboLog.kt

```diff
@@ -0,0 +1,126 @@
+package com.mattermost.turbolog
+
+import android.util.Log
+import ch.qos.logback.classic.Level
+import ch.qos.logback.classic.LoggerContext
+import ch.qos.logback.classic.encoder.PatternLayoutEncoder
+import ch.qos.logback.classic.spi.ILoggingEvent
+import ch.qos.logback.core.rolling.FixedWindowRollingPolicy
+import ch.qos.logback.core.rolling.RollingFileAppender
+import ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy
+import ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy
+import ch.qos.logback.core.util.FileSize
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.io.File
+import java.nio.charset.StandardCharsets
+
+data class ConfigureOptions(
+  val dailyRolling: Boolean? = null,
+  val logsDirectory: String? = null,
+  val maximumFileSize: Int? = null,
+  val maximumNumberOfFiles: Int? = null,
+  val logPrefix: String? = null,
+)
+
+class TurboLog {
+  companion object {
+    private val logger: Logger = LoggerFactory.getLogger(TurboLog::class.java)
+    private var logsDirectory: String? = null
+    private var configureOptions: ConfigureOptions? = null
+
+    fun getLogFiles(): Array<out File> {
+      val directory = logsDirectory?.let { File(it) }
+      return directory?.listFiles { _, name -> name.endsWith(".log") } ?: arrayOf()
+    }
+
+    fun reconfigure() {
+      configureOptions?.let { configure(it) }
+    }
+
+    fun configure(options: ConfigureOptions) {
+      val dailyRolling = options.dailyRolling ?: false
+      val maximumFileSize = options.maximumFileSize ?: (1024 * 1024)
+      val maximumNumberOfFiles = options.maximumNumberOfFiles ?: 2
+      logsDirectory = options.logsDirectory ?: ""
+      val logPrefix = options.logPrefix ?: ""
```

Comment:
default prefix should be context.packageName

Reply:
Same as my previous answer.